server/auth_function/lambda_function.py

```diff
@@ -188,7 +188,7 @@ def populate_user_if_necessary(db_connection, event) -> None:
         VALUES( {user_type_code}, {user_guid}, {cognito_user_id}, {user_name}, {business_guid},
         CURRENT_USER, CURRENT_DATE, CURRENT_USER, CURRENT_DATE)
         ON CONFLICT (user_type_code, user_guid) DO
-        UPDATE SET user_name = {user_name},  cognito_user_id = {cognito_user_id}, business_guid = {business_guid};"""
```

Comment:
Hi Basil, do you think we can still update `business_guid` using sql `CASE... WHEN`, so that if the Cognito event does provide `business_guid` for "some users" we still update the business_guid column, but if the business_guid is not present, then no update to the column?

Reply:
I just went with a simple change to minimize the need for testing, so the fix can be deployed as quickly as possible.